packages/shared/src/graphql/fragments.ts

```diff
@@ -117,11 +117,15 @@ export const COMMENT_FRAGMENT = gql`
   fragment CommentFragment on Comment {
     id
     contentHtml
+    content
     createdAt
     lastUpdatedAt
     permalink
     upvoted
     numUpvotes
+    parent {
+      id
+    }
```

Comment:
Same here, what did you need it for?
I think only comment feed uses this altogether right? and that has it's own type of fragment for this case.

Reply:
In this case, we need to know the parent comment id for both replying and edit. I think it's more a result of refactoring to a page and then back to modal. We could supply this as a prop as well if you're concerned about it.